### ISSUE TITLE
Sakai LTI 1.3 Launch Getting This Error "Missing session-id cookie issue"

### ISSUE THREAD

**Issue by anmola31** [dmitry-viskov/pylti1.3#123]
One of our clients is encountering the following error when launching the LTI 1.3 Tool. After attempting to launch the tool multiple times, it eventually loads. Instructors are experiencing this issue. Any assistance in resolving this matter would be greatly appreciated

 File "/srv/proctortrack/venv/lib/python3.6/site-packages/pylti1p3/message_launch.py", line 728, in set_launch_data_storage
 raise LtiException("Missing %s cookie" % session_cookie_name)
,
pylti1p3.exception.LtiException: Missing session-id cookie

**Comment by hmoffatt** [comment]
I have also seen "Missing session-id cookie" exceptions, but on launches from Canvas. It's not consistent and I haven't been able to reproduce it.

**Comment by NaurizAitbai** [comment]
Was it when using browser in incognito mode or blocking third-party cookies? What browser did they used?

You can use Sentry for monitoring errors and you will see who raise it, what browser used, stacktrace of exception, how many it raised, how many users raised this exception and etc.

**Comment by anmola31** [comment]
NO, they are not using incognito mode, they are using chrome browser and Multiple users are facing this issue.

**Comment by dannydotliu** [comment]
Just chiming in here, we are experiencing the same with Chrome browser (not incognito) as well and v2.0.0 of pylti1.3. From looking at the requests, it seems to be that the lti1p3-session-id cookie is not being passed through to the Canvas LTI authorisation URI `https://xx.instructure.com/api/lti/authorize_redirect?`

Cookies are marked as SameSite=None and Secure
